lib/libutils/isoc/bget_malloc.c

```diff
@@ -609,7 +613,11 @@ void *mdbg_malloc(const char *fname, int lineno, size_t size)
 {
 	struct mdbg_hdr *hdr;
 
+#if defined(ARM64)
+	COMPILE_TIME_ASSERT(sizeof(struct mdbg_hdr) == sizeof(uint32_t) * 6);
```

Comment:
Does this check for performance consideration?
If size of `struct mdbg_hdr` is properly aligned. Does it imply it would be placed at correctly aligned
memory location?